core/src/main/java/org/akaza/openclinica/service/crfdata/EnketoUrlService.java

```diff
@@ -596,5 +602,10 @@ private void addContactData(List<Bind> binds, Map<String, Object> data,StudySubj
         }
     }
 
+    public String addLeadingZeros(String s) {
+        if (s.length() >= THREAD_NAME_LENGTH) return THREAD_NAME_PREFIX+s;
+        else return THREAD_NAME_PREFIX+String.format("%0" + (THREAD_NAME_LENGTH - s.length()) + "d%s", 0, s);
```

Comment:
Just an FYI for future - You can use this method for this: https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#leftPad-java.lang.String-int-char-

Reply:
changed to StringUtils.leftPad method